src/lint/concepts.nim

```diff
@@ -36,6 +36,27 @@ proc isEveryConceptLinksFileValid*(trackDir: string): bool =
         if not isValidLinksFile(j, linksPath):
           result = false
 
+proc isValidConceptConfig(data: JsonNode, path: string): bool =
+  if isObject(data, "", path):
+    let checks = [
+      hasString(data, "blurb", path, maxLen = 350),
+      hasArrayOfStrings(data, "", "authors", path),
+      hasArrayOfStrings(data, "", "contributors", path, isRequired = false),
+    ]
+    result = allTrue(checks)
+
+proc isEveryConceptConfigValid*(trackDir: string): bool =
+  let conceptsDir = trackDir / "concepts"
+  result = true
+
+  if dirExists(conceptsDir):
+    for subdir in getSortedSubdirs(conceptsDir):
+      let configPath = subdir / ".meta" / "config.json"
+      let j = parseJsonFile(configPath, result)
+      if j != nil:
+        if not isValidConceptConfig(j, configPath):
+          result = false
```

Comment:
This implementation is very similar to the `isEveryConceptLinksFileValid` proc in the same file.

We could refactor so that we only walk over this directory once, but I think this is fine for now. If we did that, we might still want to output all the `links.json` warnings together, then all the `.meta/config.json` warnings, rather than outputting both per-exercise.

Reply:
Yeah, I'm fine with this. At some point we should decide how we want to display errors/warnings: per type of per exercise.